components/decimal/examples/code_line_diff.rs

```diff
@@ -41,6 +41,8 @@ fn main(_argc: isize, _argv: *const *const u8) -> isize {
         let decimals: (FixedDecimal, FixedDecimal) = (line.0.into(), line.1.into());
         let removed = fdf.format(&decimals.0);
         let added = fdf.format(&decimals.1);
+        assert_ne!("", removed.writeable_to_string());
```

Comment:
Question: Are these changes supposed to be here? They look unrelated.

Reply:
They're not *completely* unrelated.  I made these changes to improve the quality of the build artifacts in release mode.  It turns out that `assert_ne!()` with all the build settings does a good job at keeping things from being deleted by dead code elimination.  I could have done this in a separate PR, but I did it here because it was simpler.